modules/timepoint_list/js/timepoint_list.js

```diff
@@ -5,7 +5,7 @@ $(document).ready(function() {
     $(".timepoint_list").click(function(e) {
         e.preventDefault();
         var form = $('<form />', {
-            "action" : "main.php?test_name=imaging_browser",
+            "action" : "{$baseurl}imaging_browser/",
```

Comment:
this won't work because this is a javascript file and not a template, should isle ```loris.BaseURL``` instead